modules/html_generator.py

```diff
@@ -129,7 +129,7 @@ def generate_4chan_html(f):
 def make_thumbnail(image):
     image = image.resize((350, round(image.size[1] / image.size[0] * 350)), Image.Resampling.LANCZOS)
     if image.size[1] > 470:
-        image = ImageOps.fit(image, (350, 470), Image.ANTIALIAS)
+        image = ImageOps.fit(image, (350, 470), Image.LANCZOS)
```

Comment:
```suggestion
        image = ImageOps.fit(image, (350, 470), Image.Resampling.LANZCOS)
```

Reply:
The [documentation](https://pillow.readthedocs.io/en/stable/deprecations.html#constants) states that 

> Image.LANCZOS or Image.Resampling.LANCZOS

should be used. Is there any particular reason to go from the first variant to the second one?

Reply:
Oh, I didn’t read the documentation (whoops) No, I don’t think there should be any reason, I had just tested with what I suggested. The only reason you would choose mine would be for consistency, as line 130 uses the Resampling variation